test/sandbox_test.dart

```diff
@@ -4,5 +4,6 @@ import 'package:test/test.dart';
 void main() {
   test('calculate', () {
     expect(calculate(), 42);
+    print('test successful');
```

Comment:
We should use string concatenation
```suggestion
    print('test' + ' successful');
```